### PR TITLE
Navigation drawer: lag on older devices

### PR DESCRIPTION
**Pull request by fi3te**
Lag especially noticeable when starting another activity, e.g. startActivity(settingsIntent).
Solution: A pending runnable will be executed after the drawer has been closed.

### CODE REVIEW

**Comment by timrae** [comment]
Hi, thanks for the contribution. If I understand correctly, the intent of this PR is to make the animation more fluid? In terms of the "lag" (i.e. the time between choosing a new activity and the activity finishing loading), it should be slightly worse using the pending intent, correct?

**Comment by fi3te** [comment]
Excuse my inaccuracy :smile: Yes, the changes should make the animation more fluid. The laggy behavior was related to the Navigation Drawer which closes not fast enough and without constant speed (stuttering).
Tested with: Nexus 4 / Snapdragon S4 Pro 4x1,5 GHz

File: AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.java [hunk 1]
```diff
@@ -110,6 +117,11 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
             public void onDrawerClosed(View drawerView) {
                 super.onDrawerClosed(drawerView);
                 supportInvalidateOptionsMenu();
+
```
Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.java]: Or maybe further down, above the `pendingRunnable = new Runnable()` part... I'll leave it up to you.

Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.java]: If you could also add a second line to the commit message with similar info, that would be useful.
You can amend and force push the commit for both changes.

Comment (fi3te) [inline AnkiDroid/src/main/java/com/ichi2/anki/NavigationDrawerActivity.java]: Done :simple_smile:

**Comment by timrae** [comment]
Released in the latest alpha, it looks nice thank you! :)

**Pull request merged**